### PR TITLE
Fix: Post layout - Excerpt logic broken issue.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Trello - https://trello.com/c/kVgHs8yS/369-fix-post-layout-excerpt-logic-broken-issue

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Add a post with post content and excerpt in the excerpt box in master.
- Drag and drop the post layout any and check front-end.  it will show post content in excerpt instead of excerpt box.
- Switch to  `post-layout-excerpt-issue` and check post layout it display correctly excerpt box content if added.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

File: blocks-config/post/class-uagb-post.php [hunk 1]
```diff
@@ -1455,7 +1455,7 @@ public function render_excerpt( $attributes ) {
 			$length = ( isset( $attributes['excerptLength'] ) ) ? $attributes['excerptLength'] : 25;
 
 			if ( 'full_post' === $attributes['displayPostContentRadio'] ) {
-				$excerpt = get_the_content();
+				$excerpt = strip_shortcodes( get_the_content() );
```
Comment (vrundakansara) [inline blocks-config/post/class-uagb-post.php:1458]: We can't strip the shortcodes. Executing shortcodes is expected.

File: readme.txt [hunk 2]
```diff
@@ -168,6 +168,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 1.23.2 =
+* Fix: Post layout - Excerpt logic broken issue.
```
Comment (vrundakansara) [inline readme.txt:172]: `* Fix: Post Layouts - Showing post content instead of post except on the front-end.`

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: blocks-config/post/class-uagb-post.php (hunk 1 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php:1458]: It's showing all image galleries in the post block and it's not looking good. See here - https://share.getcloudapp.com/YEuZ1xvy

Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Updated please check

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Updated

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**